packages/synapse-react-client/src/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -4,6 +4,7 @@
 .QueryWrapperPlotNav {
   display: grid;
   position: relative;
+  grid-template-columns: 100%;
```

Comment:
fix issue where the component would extend beyond 100% width at viewport widths smaller than the 'medium' breakpoint

Reply:
Nice!